package.json

```diff
@@ -111,6 +111,11 @@
                     "type": "boolean",
                     "default": false,
                     "description": "Show the VSCode-Bazel output channel when the Bazel extension initializes."
+                },
+                "bazel.directoriesToIgnore": {
+                    "type": "string",
+                    "default": "",
+                    "description": "Regex of directors to ignore when trying to figure out if something is in a Bazel Workspace. This just impacts things that need a Bazel Workspace, BUILD/.bzl syntax and buildifier support still work."
```

Comment:
It looks like the logic below matches the entire path, not just "directories", right? Should this just be called "pathsToIgnore" then, with the following suggested description?

"Regex of paths to ignore when trying to figure out if a file is in a Bazel workspace. This impacts operations that require a workspace, like queries, but syntax highlighting and buildifier work regardless."

Reply:
done